src/App.scss

```diff
@@ -27,10 +27,18 @@
     margin-left: 1px;
 }
 
-// temp solution to fix "export as PDF" being off in the dropdown menu list
-.pf-c-button {
-    &.pf-c-dropdown__menu-item {
-        padding-left: 4px;
+// temp solution to style "Export as PDF" button
+#kebab-item-download-pdf {
+    display: block;
```

Comment:
A nit-pick: could we possibly import the styling from the PF CSS directly instead of rewriting it? SCSS syntax allows it to do easily IIRC

Reply:
Fixed